src/pages/Movie/TopInfo/ToggleToWatchlist/container.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import {
+  toggleToWatchlist as toggleToWatchlistAction
+} from '../../../../../state/movie/actions';
+import ToggleToWatchlistComponent from './component';
+
+class ToggleToWatchlistContainer extends React.Component {
+  changeWatchlistStatus = () => {
+    const {
+      id,
+      isMovieInWatchlist,
+      toggleToWatchlist
+    } = this.props;
+
+    toggleToWatchlist(id, isMovieInWatchlist);
```

Comment:
You have action with signature `toggleToWatchlistSuccess = (id, newStatus) => {...}`, so it's better to send new status as described in method params. Just send opposite value.

Reply:
Fixed